components/NavDrawer.vue

```diff
@@ -48,6 +48,10 @@ export default {
           title: 'Contributing',
           link: '/contributing',
         },
+        {
```

Comment:
Hmm do you think this would be better at the top?

It would also push down resources into a more natural spot where your eyes would see it ASAP